WordPress/gutenberg
# Add Typography support for RSS block

## What?
Add  typography support to the RSS block.
Part of: https://github.com/WordPress/gutenberg/issues/43242

## Why?
RSS block is missing typography support.

## How?
Adds the typography support via block.json

## Testing Instructions
Go to the Global Styles setting ( under appearance > editor > styles > edit styles > blocks ).
Make sure that the RSS block's  typography is configurable via Global Styles.
Verify that Global Styles are applied correctly in the editor and frontend.
Edit the template/page, Add RSS block and apply the  typography settings.
Verify that block typography styles take precedence over global block settings.
Verify that block typography display correctly in both the editor and frontend.

## Screenshots or screencast
Backend:
![backendtypo](https://github.com/user-attachments/assets/379a4de4-723a-4797-a596-9476b8fbf612)

Frontend:
![frontendtypography](https://github.com/user-attachments/assets/e93b5038-f200-4881-9f6a-6e9e270e808f)
